checks/libs/win/winpdh.py

```diff
@@ -0,0 +1,49 @@
+import time
+import sys
+import win32pdh
+import psutil
+
+class WinPDHCounter(object):
+    
+    def __init__(self, class_name, instance_name):
+        self._class_name = class_name
+        self._instance_name = instance_name
```

Comment:
Could we make this class stateless, make the functions into class methods and just mass these variables into those methods? If I'm not missing anything, it seems like you're only calling it once with each of these.